### DISCUSSION TITLE
Watch Command doesn't work unless username_filter: false

### DISCUSSION THREAD

**Discussion by hellomiakoda** [Taxel/PlexTraktSync#1753] [Support] [answered]
With the username_filter set to true, watch does nothing. It just sits there as if nothing is being watched.
If I set username_filter to false, it shows I'm playing something and is presumably sending it up to Trakt.
I deleted config and server yml files and set it up again, making really sure I picked my name from the list. 
No change, still doesn't scrobble if I set it to only scrobble me.

**Comment by glensc** [comment, marked as answer]
doesn't seem you tried to enable debug log and checked logs.

my guess is that whatever you're trying to play doesn't have trakt match. did you run inspect of the URL you tried to play?
